recipe_scrapers/chefnini.py

```diff
@@ -0,0 +1,57 @@
+# mypy: allow-untyped-defs
+
+import re
+
+from ._abstract import AbstractScraper
+
+
+class Chefnini(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "chefnini.com"
+
+    def author(self):
+        return "chefNini"
+
+    def title(self):
+        return self.soup.find("span", {"itemprop": "headline"}).get_text()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return None
+
+    def yields(self):
+        recipe_yields_text = self.soup.find(
+            "h3", {"itemprop": "recipeYield"}
+        ).get_text()
+        servings = re.match(r"Pour (\d)", recipe_yields_text).group(1)
```

Comment:
```suggestion
        servings = re.match(r"Pour (\d+)", recipe_yields_text).group(1)
```

(to allow parsing meal webpages that describe more than 9 servings)

Reply:
Yep this one is my bad, didn't plug the brain. Thanks for the review.